### ISSUE TITLE
Better demo

### ISSUE THREAD

**Issue by daviferreira** [yabwe/medium-editor#64]
We need a new demo page with on the fly feature switching and a better UI.

**Comment by j0k3r** [comment]
What could be improved on the demo page?

- a kind of menu to provide a demo for every options?
- some better use case?

**Comment by daviferreira** [comment]
Yeah, something like a sidebar with every available option and the ability to toggle them on the fly. And also yeah, a menu with different pre-defined examples/use cases would be nice.
